### ISSUE TITLE
JsonArr ambiguous constructor

### ISSUE THREAD

**Issue by piotrkot** [piotrkot/oojson#20]
Too often I get compile failure:
> reference to JsonArr is ambiguous both constructor <T>JsonArr(java.util.Collection<? extends javax.json.JsonValue>) in com.github.piotrkot.json.JsonArr and constructor <T>JsonArr(java.lang.Iterable<T>) in com.github.piotrkot.json.JsonArr match

I think this is confusing when we have two constructors so similar.

**Comment by piotrkot** [comment]
`JsonArr` was meant to replace `JsonArray`. For that it is expected to have a `JsonArr` constructor which takes `JsonArray` parameter. And that is the link between old library and this library. I personally don't think we should make more connections between old and new libraries. So arguments accepted by `JsonArray` constructor do not have to be accepted as arguments to the `JsonArr` constructors.

But if you really have to handle them please use
```
        final ListOf<JsonValue> values = new ListOf<>(
            Json.createValue(5.5),
            Json.createValue("42"),
            JsonValue.TRUE
        );
        MatcherAssert.assertThat(
            new JsonArr<>(
                new Mapped<>(
                    value -> new ObjectFound(value).asObject(),
                    values
                )
            ).jsonValue().toString(),
            Matchers.is("[5.5,\"42\",true]")
        );
```

**Comment by piotrkot** [comment]
Fixed with version 1.6.1
